src/models/m_config.erl

```diff
@@ -176,6 +225,11 @@ delete(Module, Key, Context) ->
     z_db:q("delete from config where module = $1 and key = $2", [Module, Key], Context),
     z_depcache:flush(config, Context),
     z_notifier:notify(#m_config_update{module=Module, key=Key, value=undefined}, Context),
+    z:warning(
```

Comment:
Why `warning`? This should be `info` as well, as `warning` indicates something is wrong.

Reply:
I was thinking that it is a config change, and more important than the other events. Same for modules.

Reply:
info it is.